README.md

```diff
@@ -211,9 +211,16 @@ First, start postgres with:
 $ npm run postgres
 ```
 
-Then in another terminal:
+Then you can, in another terminal, find the running docker, init the DB, then run the tests:
 
 ```
+$ docker ps
+CONTAINER ID        IMAGE                 COMMAND                  CREATED             STATUS              PORTS                    NAMES
+28341f85c4cd        postgres:9.6-alpine   "docker-entrypoint.s…"   3 minutes ago       Up 3 minutes        0.0.0.0:5432->5432/tcp   jovial_shockley
+
+$ docker exec -ti jovial_shockley /usr/local/bin/psql -d postgres -U postgres  -c 'CREATE TABLE users(id serial PRIMARY KEY, username VARCHAR (50) NOT NULL);'
+CREATE TABLE
```

Comment:
@temsa just noticed, is this left over by any chance?

Reply:
@cemremengu it is not, is is just the output from the `psql` command, like the `docker ps` one above it :)

Reply:
Thanks, I need more coffee :)